src/mono/mono/component/debugger-agent.c

```diff
@@ -661,7 +660,7 @@ debugger_agent_parse_options (char *options)
 		} else if (strncmp (arg, "timeout=", 8) == 0) {
 			agent_config.timeout = atoi (arg + 8);
 		} else if (strncmp (arg, "launch=", 7) == 0) {
-			agent_config.launch = g_strdup (arg + 7);
+			// no longer supported
```

Comment:
I talked to Thays and she told me it was never actually used anywhere, I just kept it to "document" that this was an option in the past.

Reply:
No idea what this option was used for, it was added in 2009, I think it might be used to start some kind of tcp tunnel so the debugger and debuggee can connect.